docs/installation.md

```diff
@@ -2,422 +2,437 @@
+* FreeBSD 
 * Ubuntu
 
 
-### *Database engine*
+### 1.2 *Database engine*
 
-Zonemaster *Backend* needs a database engine.
-This guide provides instructions for the following ones:
+Zonemaster *Backend* needs a database engine.  The choice for the database are
+as follows :
 
-* MySQL
-* PostgreSQL (currently no instructions for CentOS)
+* MySQL 
+* PostgreSQL (currently no instructions for CentOS) 
 * SQLite (currently no instructions for any OS)
 
 
-## Prerequisites
+## 2. Prerequisites
 
-This guide assumes that the following softwares are already installed on the target system:
+This guide assumes that the following softwares are already installed on the
+target system:
 
-* the chosen operating system
-* sudo (only for installation and other administrative tasks)
-* cpanm (only for installation)
+* the chosen operating system sudo (only for installation and other
+  administrative tasks) 
+* cpanm (only for installation) 
 * curl (only for post-installation sanity check)
+* [Zonemaster Engine](https://github.com/dotse/zonemaster-engine/blob/master/docs/installation.md) is installed 
 
+## 3. Dependencies
 
-## Dependencies
-
-This section shows you how to install all software that your Zonemaster *Backend* will depend on.
-Find and follow the subsection relevant for your chosen operating system below.
+This section shows you how to install all software that your Zonemaster
+*Backend* will depend on.  Find and follow the subsection relevant for your
+chosen operating system below.
 
 
-### *Installing dependencies on CentOS*
+### 3.1 Installing dependencies on CentOS
 
-Unless you've already done so, install Zonemaster *Engine* according to its own [installtion instruction](https://github.com/dotse/zonemaster-engine/blob/master/docs/installation.md).
-
-```sh
-sudo yum install perl-Module-Install perl-IO-CaptureOutput perl-String-ShellQuote
-sudo cpanm -i Config::IniFiles Daemon::Control JSON::RPC::Dispatch Parallel::ForkManager Plack::Builder Plack::Middleware::Debug Router::Simple::Declare Starman
+```sh 
+sudo yum install perl-Module-Install perl-IO-CaptureOutput perl-String-ShellQuote sudo cpanm -i Config::IniFiles Daemon::Control JSON::RPC::Dispatch Parallel::ForkManager Plack::Builder Plack::Middleware::Debug Router::Simple::Declare Starman 
 ```
 
-Install the chosen database engine and related dependencies, and start the database engine:
+Install the chosen database engine and related dependencies.
 
-* MySQL
+#### 3.1.1 MySQL
 
-  ```sh
-  sudo yum install wget
-  wget http://repo.mysql.com/mysql-community-release-el7-5.noarch.rpm
-  sudo rpm -ivh mysql-community-release-el7-5.noarch.rpm
-  sudo yum install mysql-server perl-DBD-mysql
-  sudo systemctl start mysqld
-  ```
-
-* PostgreSQL
+```sh 
+sudo yum install wget wget http://repo.mysql.com/mysql-community-release-el7-5.noarch.rpm 
+sudo rpm -ivh mysql-community-release-el7-5.noarch.rpm 
+sudo yum install mysql-server perl-DBD-mysql 
+sudo systemctl start mysqld 
+```
 

```

Comment:
It does not look correct. You have collided the first two lines. It should be

```sh
sudo yum install wget
wget http://repo.mysql.com/mysql-community-release-el7-5.noarch.rpm
sudo rpm -ivh mysql-community-release-el7-5.noarch.rpm
sudo yum install mysql-server perl-DBD-mysql
sudo systemctl start mysqld
```
as far as I can see.